src/panels/UserPanelsPanel/UserPanelsPanel.tsx

```diff
@@ -140,20 +144,23 @@ export function UserPanelsPanel() {
               onClick={addWebPanel}
               disabled={!panelURL}
               size={'lg'}
-              aria-label={'Open web panel'}
+              aria-label={t('web-panels.url.aria-label')}
             >
               <OpenWindowIcon />
             </ActionIcon>
           }
         />
       </Group>
       <Title mt={'xs'} mb={'xs'} order={3}>
-        Recently Opened URLs
+        {t('recently-opened-panels.title')}
       </Title>
```

Comment:
The name "Recently Opened URLs" is a bit mismatch now that the hub is in there by default. However, I don't know what one would change it to, since it's not stored between runs... 

Maybe the hub should really be added to a separate list and not be part of the recently opened URLs? Or, I could also be overthinking this